src/tests/ftest/launch.py

```diff
@@ -556,13 +557,14 @@ def get_test_list(tags):
     return test_tags, test_list
 
 
-def get_test_files(test_list, args, yaml_dir):
+def get_test_files(test_list, args, yaml_dir, vmd_flag=False):
```

Comment:
We cannot accurately determine ahead of making any keyword replacements as to whether or not to set the `DAOS_DISABLE_VMD` env var.  As we originally discussed this should be a list of vmd addresses.  With this list we can accurately determine the `DAOS_DISABLE_VMD` setting based upon whether or not we replaced a `bdev_list` entry with a vmd device.  There may be situations where multiple tests match the specified tags and one may not use NVMe - in this case we don't want to set `DAOS_DISABLE_VMD` to `False`.

Reply:
I am not clear... There is no need to know the VMD address. I believe this if condition should handle well... Am I missing something?

    if args.nvme == "auto_vmd":
        command_list = [
            "/sbin/lspci -D", "grep 'Volume Management Device NVMe'"]
        vmd_include_flag = True
    elif args.nvme == "auto_vmd_disks":
        command_list = [
            "/sbin/lspci -D",
            "grep -E '^[0-9a-f]{5}:[0-9a-f]{2}:[0-9a-f]{2}.[0-9a-f] "
            "Non-Volatile memory controller:'"]
    elif args.nvme == "auto_vmd_nvme":
        command_list = [
            "/sbin/lspci -D",
            "grep -E '^[0-9a-f]{4,5}:[0-9a-f]{2}:[0-9a-f]{2}.[0-9a-f] "
            "Non-Volatile memory controller:'"]
    elif args.nvme == "auto_vmd_mixed":
        command_list1 = [
            "/sbin/lspci -D",
            "grep -E '^[0-9a-f]{4}:[0-9a-f]{2}:[0-9a-f]{2}.[0-9a-f] "
            "Non-Volatile memory controller:'"]
        command_list2 = [
            "/sbin/lspci -D", "grep 'Volume Management Device NVMe'"]
        vmd_include_flag = True